src/backend/metal/src/internal.rs

```diff
@@ -200,11 +200,11 @@ impl DepthStencilStates {
     }
 
     // TODO: avoid locking for writes every time
-    pub fn get<'a>(
-        &'a self,
+    pub fn get(
+        &self,
         desc: pso::DepthStencilDesc,
         device: &Mutex<metal::Device>,
```

Comment:
doesn't the self lifetime now affect this parameter as well?

Reply:
According to [Clippy.needless_lifetimes](https://rust-lang-nursery.github.io/rust-clippy/master/index.html#needless_lifetimes) we can simplify 'a lifetime marker.